docs/reference/modules/scripting.asciidoc

```diff
@@ -24,23 +24,18 @@ and give the most flexibility.
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Language
-    |Sandboxed
     |Required plugin
```

Comment:
Hmm, deleting the `Sandboxed` column feels like we're losing important security information, which should be trumpeted right from the first page.  I thought of changing the column title to `Safe`, but perhaps `Sandboxed` is a better description?

Either way, I think this column should remain.